docs/data/data-grid/column-definition/RenderExpandCellGrid.tsx

```diff
@@ -108,7 +108,7 @@ const GridCellExpand = React.memo(function GridCellExpand(
   );
 });
 
-function renderCellExpand(params: GridRenderCellParams<string>) {
+function renderCellExpand(params: GridRenderCellParams) {
```

Comment:
Shouldn't it be 

```suggestion
function renderCellExpand(params: GridRenderCellParams<unknown, string>) {
```

or 

```suggestion
function renderCellExpand(params: GridRenderCellParams<any, string>) {
```

And in `RenderCellGrid` you missed a change I think (causing the proptype change) :+1:

Reply:
The `RenderCellGrid.tsx` still needs to be updated:

```diff
-GridRenderCellParams<Date>
+GridRenderCellParams<any, Date>
```